pythonforandroid/build.py

```diff
@@ -350,20 +350,7 @@ def prepare_build_environment(self,
                     'set it with `--ndk-version=...`.')
         self.ndk_ver = ndk_ver
 
-        ndk_api = None
-        if user_ndk_api:
-            ndk_api = user_ndk_api
-            info('Getting NDK API version (i.e. minimum supported API) from user argument')
-        elif 'NDKAPI' in environ:
-            ndk_api = environ.get('NDKAPI', None)
-            info('Found Android API target in $NDKAPI')
-        else:
-            ndk_api = min(self.android_api, DEFAULT_NDK_API)
-            warning('NDK API target was not set manually, using '
-                    'the default of {} = min(android-api={}, default ndk-api={})'.format(
-                        ndk_api, self.android_api, DEFAULT_NDK_API))
-        ndk_api = int(ndk_api)
-        self.ndk_api = ndk_api
+        self.ndk_api = user_ndk_api
```

Comment:
Can we possibly keep NDKAPI env var support in? I think this is really important for all options that are more dependent on the installed build env (NDK/SDK/... downloaded) rather than the project itself - because otherwise, these can't be preconfigured to proper defaults in e.g. a docker build env